File: src/main/java/com/patient/treatment/documentation/gui/model/dto/DocumentationInterface.java
package com.patient.treatment.documentation.gui.model.dto;

import com.patient.treatment.documentation.gui.model.entites.Doctor;
import com.patient.treatment.documentation.gui.model.entites.Patient;

public interface DocumentationInterface {

    String getInterview();

    void setInterview(String interview);

    String getPhysicalExamination();

    void setPhysicalExamination(String physicalExamination);

    String getDiagnosisOfTheDisease();

    void setDiagnosisOfTheDisease(String diagnosisOfTheDisease);

    String getRecommendations();

    void setRecommendations(String recommendations);

    String getMedicines();

    void setMedicines(String medicines);

    Doctor getDoctor();

    void setDoctor(Doctor doctor);

    Patient getPatient();

    void setPatient(Patient patient);

}

File: src/main/web/src/app/model/documentation-interface.ts
export interface DocumentationFormInterface {

  id: number

  interview: string;

  physicalExamination: string;

  diagnosisOfTheDisease: string;

  recommendations: string;

  medicines: string;

  creationDate: Date;

}

export class Documentation implements DocumentationFormInterface {
  id: number;
  diagnosisOfTheDisease: string;
  interview: string;
  medicines: string;
  physicalExamination: string;
  recommendations: string;
  creationDate: Date;
}
